utils/github.go

```diff
@@ -131,3 +138,14 @@ func CreateComment(client *github.Client, owner, repoName, message string, pull
 
 	return nil
 }
+
+// GetPullRequestDetails creates a output for the action depending on the values on the public repository and github team name validation
+func Results(client *github.Client, team, public bool) error {
+	if public && team {
+		githubactions.SetOutput("valid", "true")
+		return nil
+	} else {
```

Comment:
Unnecessary else block

```suggestion
	}
	
	githubactions.SetOutput("valid", "false")
	return fmt.Errorf("repository is not public or team name is invalid")
```